src/globus_sdk/services/groups/client.py

```diff
@@ -13,3 +166,157 @@ class GroupsClient(client.BaseClient):
+        return self.get(f"/groups/{group_id}", params=params)
+
+    def delete_group(self, group_id: str):
+        return self.delete(f"/groups/{group_id}")
+
+    def create_group(self, name: str, parent_id: Optional[str] = None):
+        """
+        Create a group with the given name.  If a parent id is included, the
+        group will be a subgroup of the given parent group.
+        """
+        data = {"name": name, "parent_id": parent_id}
+        return self.post("/groups", data=data)
+
+    def get_group_policies(self, group_id: str):
+        """
+        Get policies for the given group
+        """
+        return self.get(f"/groups/{group_id}/policies")
+
+    def set_group_policies(
+        self,
+        group_id: str,
+        is_high_assurance: bool,
+        group_visibility: GroupVisibility,
+        group_members_visibility: GroupMemberVisibility,
+        join_requests: bool,
+        signup_fields: Sequence[RequiredSignupFields],
+        authentication_assurance_timeout: Optional[int] = None,
+    ):
+        data = {
+            "is_high_assurance": is_high_assurance,
+            "group_visibility": group_visibility.value,
+            "group_members_visibility": str(group_members_visibility),
+            "join_requests": join_requests,
+            "signup_fields": [field.value for field in signup_fields],
+        }
+        if authentication_assurance_timeout:
+            data["authentication_assurance_timeout"] = authentication_assurance_timeout
+
+        return self.put(f"/groups/{group_id}/policies", data=data)
+
+    def get_identity_preferences(self):
+        """
+        Get identity preferences.  Currently this only includes whether the
+        user allows themselves to be added to groups.
+        """
+        return self.get("/preferences")
+
+    def set_identity_preferences(self, preferences: dict):
+        """
+        Set identity preferences.  Currently this only includes whether the
+        user allows themselves to be added to groups.
+        """
+        return self.put("/preferences", data=preferences)
+
+    def get_membership_fields(self, group_id: str):
+        return self.get(f"/groups/{group_id}/membership_fields")
+
+    def set_membership_fields(self, group_id: str, membership_fields: dict):
+        return self.put(f"/groups/{group_id}/membership_fields", data=membership_fields)
+
+    def batch_membership_action(self, group_id, actions: BatchMembershipActions):
+        """
+        Execute a batch of actions against several group memberships.
+        """
+        return self.post(f"/groups/{group_id}", data=actions.to_dict())
+
+    def accept_invite(self, group_id: str, identity_id: str):
+        """
+        Accept invite for an identity.  The identity must belong to
+        the identity set of the authenticated user.
+        """
+        actions = BatchMembershipActions().accept_invites([identity_id])
+        return self.batch_membership_action(group_id, actions)
```

Comment:
I'm torn about the inclusion of these wrapper methods. They will certainly be convenient, but they seem to duplicate a lot -- especially the docstrings -- of relevant `BatchMembershipActions` methods.

I'm wondering if long-term maintenance would be better served, especially as things change and new parameters get added over time, by not having these and expecting usage to always be of the form
```python
client.batch_membership_action(
    groupid,
    BatchMembershipActions().accept_invites([id1, id2]),
)
```

Simply because we have limited resources for maintenance, I am somewhat concerned about drift and staleness. These projects are receiving attention right now, but in general they tend to be pushed down in priority.